recipes/coin3d/meta.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+    name: coin3d
+    version: 4.0.0
+
+source:
+    fn: coin.tar.gz
+    url: https://github.com/FreeCAD/pivy/releases/download/0.6.3/Coin3D-coin-8e8337f61b05.tar.gz
```

Comment:
You might want to use some of the jinja formatting from the example, it may make things easier https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L4

Reply:
thanks for having a look at this. Updated: https://github.com/conda-forge/staged-recipes/pull/4732/commits/109c50adbc163a2b6739823cb61aafab6159d32d